examples/data-services/src/test/java/com/acme/DataServiceTest.java

```diff
@@ -0,0 +1,33 @@
+package com.acme;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.DatabaseClientFactory;
+import com.marklogic.client.DatabaseClientFactory.SecurityContext;
+
+public class DataServiceTest {
+  @Test
+  public void testMockWhatsUp() {
+    DatabaseClient db = null;
+    assertEquals("Hey! Hey!", HelloWorldMock.on(db).whatsUp("Hey!", 2L));
+  }
+
+  @Test
+  public void testWhatsUp() {
+    // Note: The user must have the `rest-extension-user` role to be able to
```

Comment:
I don’t think this is actually true. Removing this may simplify the overall setup.